### PR TITLE
feat: [CHAOS-1043]: Decreasing chaos license limits to 2 infra and 30 wfruns/month

### PR DESCRIPTION
**Pull request by imrajdas**
## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>

You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
    - CodeFormat: `trigger codeformat`
    - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/42737)

### CODE REVIEW

File: 930-ng-license-manager/src/main/java/io/harness/licensing/interfaces/clients/local/ChaosLocalClient.java [hunk 1]
```diff
@@ -24,9 +24,9 @@
 public class ChaosLocalClient implements ChaosModuleLicenseClient {
   private static final int ENTERPRISE_TRIAL_CHAOS_EXPERIMENT_RUNS = 10000;
   private static final int TEAM_TRIAL_CHAOS_EXPERIMENT_RUNS = 10000;
-  private static final int FREE_TRIAL_CHAOS_EXPERIMENT_RUNS = 10000;
-
-  private static final int FREE_TRIAL_CHAOS_INFRASTRUCTURES = 1000;
+  private static final int TEAM_TRIAL_CHAOS_INFRASTRUCTURES = 1000;
+  private static final int FREE_TRIAL_CHAOS_EXPERIMENT_RUNS = 30;
+  private static final int FREE_TRIAL_CHAOS_INFRASTRUCTURES = 2;
```
Comment (arkajyotiMukherjee) [inline 930-ng-license-manager/src/main/java/io/harness/licensing/interfaces/clients/local/ChaosLocalClient.java:29]: I think Free and Team plans don't have a trial. Only enterprise has a trial.

Comment (imrajdas) [reply 930-ng-license-manager/src/main/java/io/harness/licensing/interfaces/clients/local/ChaosLocalClient.java]: done

**Comment by imrajdas** [comment]
Screenshot 2023-01-18 at 12 34 55 PM
Screenshot 2023-01-18 at 12 24 56 PM
Screenshot 2023-01-18 at 12 29 18 PM

**Review by arkajyotiMukherjee — APPROVED** [review]

**Review by arkajyotiMukherjee — COMMENTED** [review]
> Reviewed 1 of 1 files at r1, 1 of 1 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/42737)* status: 1. unresolved discussions.

**Comment by imrajdas** [comment]
https://harness.slack.com/archives/C04AS79PXQ9/p1674024156413449?thread_ts=1673872692.582169&cid=C04AS79PXQ9

**Review by arkajyotiMukherjee — DISMISSED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/42737)* status: :shipit: complete! ✓

**Review by arkajyotiMukherjee — APPROVED** [review]
> Reviewed 1 of 1 files at r5, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/42737)* status: :shipit: complete! ✓

**Comment by imrajdas** [comment]
trigger ti1

**Pull request merged**